<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>397</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="corrections_listWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>191</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Коррекции</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_cor_but">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_exist_cor_but">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_but">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ew</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_but">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ew_lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="cor_textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Комментарий к коррекции</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
